### DISCUSSION TITLE
Support doubleSided gltf export

### DISCUSSION THREAD

**Discussion by Jimmy-KL** [pyvista/pyvista#4031] [Ideas]
The gltf model exported from pyvista using the method `BasePlotter.export_gltf` is not double-sided rendering. It's OK if we don't look inside a closed surface. But if we want to look inside, for example, wandering in a tunnel, or it's an open surface like a terrain, we look from the bottom, we need double-sided rendering. 
If we export the pyvista example random_hills as gltf model, we take look from the downside, we cannot see the hill.
 The upside:
![image](https://user-images.githubusercontent.com/114913030/219113016-23997256-1e29-46c4-ab29-6dc2b8877818.png)
The downside:
![image](https://user-images.githubusercontent.com/114913030/219112951-783997f2-0adb-4a93-8c7d-71920a60a116.png)
If we add a parameter `"doubleSided": true ` to the materials parameter in gltf model, we can see the model from both sides.
![image](https://user-images.githubusercontent.com/114913030/219114052-88ff39cd-ebd6-4c1a-bb0f-afb43b366ece.png)
![image](https://user-images.githubusercontent.com/114913030/219113932-d3b3d0b0-d489-4d42-9b5a-46ee438ae11e.png)
So I think maybe we can add an argument `double_sided` to the method `BasePlotter.export_glt`, so that users can export double-sided gltf model if they want.

**Comment by banesullivan** [comment]
PyVista relies on VTK's [`vtkGLTFExporter`](https://vtk.org/doc/nightly/html/classvtkGLTFExporter.html) for the GLTF export. I don't immediately see an option exposed on that class to control this, so I may recommend you raise an issue with VTK to support this configuration: https://gitlab.kitware.com/vtk/vtk

**Comment by Jimmy-KL** [comment]
@banesullivan OK, I will report this issue to VTK. I checked `vtkGLTFExport`, but there is no such option. I also tried Paraview, but no such option either.

**Comment by Jimmy-KL** [comment]
Here is the issue I reported for vtk: https://gitlab.kitware.com/vtk/vtk/-/issues/18827
